src/components/PcpFeed/PcpCard/index.js

```diff
@@ -0,0 +1,114 @@
+import * as moment from "moment-timezone";
+import PropTypes from "prop-types";
+import { makeStyles } from "tss-react/mui";
+
+const useStyles = makeStyles()((theme) => ({
+	container: {
+		color: theme.palette.text.primary,
+		background: "white",
+		padding: "0.6rem 0 0.6rem 0",
+		display: "flex",
+		gap: "0.625rem",
+		flexDirection: "row",
+		width: "100%",
+		borderBottom: "1px solid #585858",
+	},
+	card: {
+		width: "100%",
+	},
+	header: {
+		fontWeight: 700,
+	},
+	content: {
+		display: "flex",
+		flexDirection: "row",
+		justifyContent: "space-between",
+		flexWrap: "wrap",
+	},
+	status: {
+		display: "inline-flex",
+		marginLeft: "auto",
+		"& div": {
+			fontWeight: 700,
+			padding: "0.4rem 0.875rem 0.4rem 0.875rem",
+			borderRadius: "1rem",
+			fontSize: "0.875rem",
+			lineHeight: "1rem",
+			color: "white",
+			display: "flex",
+			justifyContent: "center",
+			width: "5.624rem",
```

Comment:
typo?

Reply:
No, this is based on the figma where the status badge is a fixed size. Should this be done differently?

Reply:
oh haha yea, the figma doesn't have "Upcoming" which is longer than the other status', but the weird rem was a mistake. Updated in 18da33b.